domains/sandun.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Portfolio website of Sandun Madhushan",
+  "repo": "https://github.com/sandunMadhushan/sandunMadhushan.github.io",
+  "owner": {
+    "username": "sandunMadhushan",
+    "email": "sandunhmadushan9330@gmail.com"
+  },
+  "record": {
+    "CNAME": "sandunMadhushan.github.io",
+    "TXT": "ae3c9fbe5a954f90454356b05ad59b"
```

Comment:
Invalid GitHub Page file verification name. See docs for more information: https://docs.is-a.dev/guides/github-pages

Make sure to also include a GitHub verification record in another file. You can find out more information on this at the docs: https://docs.is-a.dev/guides/github-pages